eng/test-job.yml

```diff
@@ -203,6 +203,31 @@ jobs:
           - jitstress2_tiered
           - zapdisable
           - tailcallstress
+        ${{ if eq(parameters.testGroup, 'outerloop-jitstressisas') }}:
+          scenarios:
+          - jitstressisas_incompletehwintrinsic
+          - jitstressisas_nohwintrinsic
+          - jitstressisas_nohwintrinsic_nosimd
+          - jitstressisas_nosimd
+        ${{ if and(eq(parameters.testGroup, 'outerloop-jitstressisas'), or(eq(parameters.archType, 'x64'), eq(parameters.archType, 'x86'))) }}:
+          scenarios:
+          - jitstressisas_noaes
```

Comment:
There is no point in wasting ARM/ARM64 machines running the x86/x64 configuration switches. Likewise, I see no reason to run the ARM/ARM64 specific switches on x86/x64 once we start ramping those up.

Reply:
This is the only major error I've seen in your YAML: you redefined scenarios here